src/modules/auth/services/auth.service.ts

```
@@ -0,0 +1,65 @@
+
+    // Use the HashingService to compare the password
+    const isValid = await this.hashingService.comparePasswords(
+      password,
+      user.password,
+    );
+    if (!isValid) {
+      throw new UnauthorizedException('Invalid credentials');
+    }
+
+    // Exclude the password field before returning
+    const { password: userPassword, ...userWithoutPassword } = user;
```

ESLint

[error] 'userPassword' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)